dist/app.html

```diff
@@ -13,6 +13,9 @@
         width: 100%;
         background-color: rgb(53, 59, 70);
       }
+      * {
+        outline: none;
```

Comment:
haha, agree that the outline around the slider monitor looks ugly, but it's for accessibility reason, so you can move between buttons and slider by pressing Tab. Otherwise you don't know what's selected.